### DISCUSSION TITLE
Unable to enable custom domain names properly

### DISCUSSION THREAD

**Discussion by zukhouzi** [community/community#112575] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

The domain name DNS has resolved the A record and CNAME, and the custom domain name settings on the Github page have also been saved with a prompt of "DNS check successful". Accessing the custom domain name can be accessed normally and the website content can be displayed. But after a few minutes, it prompted 404. I don't know what's going on! Please advise, thank you!

Custom domain name：www.nice.im  github page: zukhouzi.github.io

**Comment by queenofcorgis** [comment, marked as answer]
Hi @zukhouzi , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be closing and locking any posts in languages other than English for now, including this one.
